config/initializers/2_limited_federation_mode.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+
+Rails.application.configure do
+  config.x.limited_federation_mode = (ENV['LIMITED_FEDERATION_MODE'] || ENV['WHITELIST_MODE']) == 'true'
+
+  warn 'WARN: The environment variable WHITELIST_MODE has been replaced with LIMITED_FEDERATION_MODE, you should rename this environment variable in your configuration.' if ENV.has_key('WHITELIST_MODE')
```

Comment:
@ClearlyClaire looks like it's failing due to this, I'll have to look up whether `ENV.has_key?` is a recent addition..

Reply:
Okay, I see the mistake. I have a feeling I shouldn't have been writing code last week :|